app_server/app/util/Utility.js

```diff
@@ -60,3 +61,62 @@ exports.createLogger = function (filename) {
     ]
   });
 };
+
+var clearUserProfile =
```

Comment:
Alright. Would it be better making another file for it, though? Utility honestly sounds too generic for these methods; that's why I asked in the first place.